### ISSUE TITLE
docker.sh fails: "urls must start with a leading slash"

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#101]
This is only a problem starting from docker.sh: quick-start.sh still works fine. Will try reverting to find the commit that caused the issue.

```
Traceback (most recent call last):
  File "./app/main.py", line 49, in <module>
    app = create_app()
  File "./app/main.py", line 33, in create_app
    app.register_blueprint(routes_markdown.blueprint)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 98, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1167, in register_blueprint
    blueprint.register(self, options, first_registration)
  File "/usr/local/lib/python3.7/site-packages/flask/blueprints.py", line 256, in register
    deferred(state)
  File "/usr/local/lib/python3.7/site-packages/flask/blueprints.py", line 294, in <lambda>
    self.record(lambda s: s.add_url_rule(rule, endpoint, view_func, **options))
  File "/usr/local/lib/python3.7/site-packages/flask/blueprints.py", line 86, in add_url_rule
    **options
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 98, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1274, in add_url_rule
    rule = self.url_rule_class(rule, methods=methods, **options)
  File "/usr/local/lib/python3.7/site-packages/werkzeug/routing.py", line 641, in __init__
    raise ValueError("urls must start with a leading slash")
ValueError: urls must start with a leading slash
unable to load app 0 (mountpoint='') (callable not found or import error)
```

**Comment by mccalluc** [comment]
It broke in pr #97

**Comment by mccalluc** [comment]
Weird: Removing context/app/schemas/README.md seems to fix it?

I want to understand what's going on, and add a test that would catch this.

**Comment by mccalluc** [comment]
Fixed in #103.
